roles/user-liaison.md

```diff
@@ -0,0 +1,10 @@
+# User Liaison
+The user liaison is a role within the Saturn team that addresses issues that block only a few users on prod (whereas on-call addresses issues that block many users on prod) The only requirements to participate in the user liaison process are having a development setup and access to the #workbench-userliaison channel. This channel is private, so to take on user liaison duties, message a member of the comms team. The user liaison is responsible for watching #saturn and #workbench-userliaison for any reports on bugs in our services.
+
+## Typical flow
+A user issue comes in through one of the channels. The user liaison jumps on to find a root cause, then either:
+* If the issue is large, make a bug ticket to bring in later, and let the reporter know you have filed a ticket
+* If the issue is small enough, file a ticket to address immediately, bring the ticket into the sprint, and let the team (and especially product owner) know
+
+## Why we have the role
+The user liaison's job is to be a buffer against the world for the team. However, if the UL needs help, they can and should ask the team. (Their role is to handle initial triage, but in the spirit of the UL not needing to have all the answers, the team is encouraged to lift and support the UL)
```

Comment:
In addition to protecting the rest of the team, the UL also provides benefit to the front-line support team as a single point of contact, who is interruptible by default (during their rotation).